src/content/api/compilation-hooks.md

```diff
@@ -161,7 +161,7 @@ W> This hook will be removed in v5.0.0
 
 `SyncBailHook`
 
-Called at the beginning of the modules optimization phase. A plugin can tap into this hook to perform modules optimizations.
+Called at the beginning of the modules optimization phase. A plugin can tap into this hook to perform module optimizations.
```

Comment:
I think its actually `modules` because in the callback you receive many of them

Reply:
Similar as below.
```suggestion
Called at the beginning of the module optimization phase. A plugin can tap into this hook to perform optimizations on modules.
```